ERCS/ERC-271.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: 271
+title: SHIELD Security Token Standard (ERC-271)
```

Comment:
```suggestion
title: Non-fungible Security Token
```
ERCs are not for branding/marketing.  Don't include product names in titles.
Standard is superfluous, as is the ERC number in titles.

Title should be descriptive.  Note: I am not an ERC editor